quantum/quantum_keycodes.h

```diff
@@ -71,6 +71,12 @@ enum quantum_keycodes {
     QK_TAP_DANCE_MAX      = 0x57FF,
     QK_LAYER_TAP_TOGGLE   = 0x5800,
     QK_LAYER_TAP_TOGGLE_MAX = 0x58FF,
+#ifdef STENO_ENABLE
```

Comment:
:) I think we'll just go by whoever opens stuff first in the future, unless we have reason not to. I think this works well in the higher ranges, since you can turn it off.